### DISCUSSION TITLE
Webshare Proxies

### DISCUSSION THREAD

**Discussion by Cyx-Development** [KevinBytesTheDust/CVAmp#270] [General]
Hey there,
as Webshare Proxies frequently get blocked by too many requests to passport.twich.tv, is it possible to just block requests to that address when not using authentication anyway?

**Comment by YOIAE** [comment]
> Hey there, as Webshare Proxies frequently get blocked by too many requests to passport.twich.tv, is it possible to just block requests to that address when not using authentication anyway?

how did you reply webshare verification question...same problem with me....

**Comment by jlplenio** [comment]
Webshare told me they will increase the limit, so it should not be hit by normal use.

> Hey there, as Webshare Proxies frequently get blocked by too many requests to passport.twich.tv, is it possible to just block requests to that address when not using authentication anyway?

Yes, can do that. But I will have to investigate, if this breaks the streaming and or if the viewer is counted.

**Reply by Cyx-Development** [reply]
Not sure about that. Webshare completly blocked my account now and cancelled all services i ordered there
